composer.json

```diff
@@ -1,10 +1,75 @@
 {
-  "name": "su-sws/stanford_publication",
-  "description": "A Drupal feature.",
-  "type": "drupal-custom-module",
-  "license": "GPL-2.0+",
-  "minimum-stability": "dev",
-  "require": {
-      "composer/installers": "^1.2"
-  }
+    "name": "su-sws/stanford_publication",
+    "description": "A Drupal publication content type and views.",
+    "type": "drupal-custom-module",
+    "license": "GPL-2.0-or-later",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/locales",
+                "version": "1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/locales.git",
+                    "reference": "master"
+                }
+            }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/styles-distribution",
+                "version": "1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/styles-distribution.git",
+                    "reference": "master"
+                }
+            }
+        }
+    ],
+    "require": {
+        "citation-style-language/locales": "1.0.0",
+        "citation-style-language/styles-distribution": "1.0.0",
+        "composer/installers": "^1.2",
+        "drupal/element_class_formatter": "^1.3",
+        "drupal/field_formatter_class": "^1.0",
+        "drupal/field_group": "^3.0",
+        "drupal/inline_entity_form": "^1.0",
+        "drupal/name": "^1.0",
```

Comment:
This is a fancy field UI but it is only in `rc3`. It doesn't look as actively developed. Have you read through the code? Any concerns?

Reply:
I've done a high level review. It really makes name fields much easier especially with the way it constructs the keyed value of the field item.

Reply:
@imonroe please have a quick pass over this module's code to call out any red flags.